src/main/java/frc/robot/Robot.java

```diff
@@ -50,6 +51,7 @@ public static enum LedEnum {
   public static final boolean intakeEnabled = true;
   public static final boolean swerveEnabled = true;
   public static final boolean armEnabled = true;
+  public static final boolean offsetsFixed = true;
```

Comment:
Separate  this from the subsystem enables, since it serves a different function

Reply:
done, put in its own area